spec/support/acceptance_helper.rb

```diff
@@ -218,7 +218,7 @@ def _add_changes(type, changes, dst)
     dst[type].sort!
 
   rescue RuntimeError => e
-    raise unless e.message == "can't modify frozen Hash"
+    raise unless /can't modify frozen(?: instance of)? Hash/ =~ e.message
```

Comment:
I'm not sure why we need this in the first place? It seems we `.dup` each `change` in `#_relative_path`: `unfrozen_copy = change.dup` so that case shouldn't actually happen?